src/Hooks/useBrands.js

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const useBrands = (url) => {
```

Comment:
I think you dont need to recive url in each hook because each hook is specific for each request.